src/System.Runtime/tests/System/DateTime.cs

```diff
@@ -291,6 +294,78 @@ public static void TestTryParseExactA()
         Assert.Equal(s, actual);
     }
 
+    [Fact]
+    [ActiveIssue(846, PlatformID.AnyUnix)]
+    public static void TestGetDateTimeFormats()
+    {
+        var savedCulture = CultureInfo.CurrentCulture;
```

Comment:
Also we don't need to set the current culture. we can format the dates with specific culture (like en-US) without changing the current culture.